packages/server/src/lib/ab.ts

```diff
@@ -66,31 +61,40 @@ export const selectVariant = <V extends Variant, T extends Test<V>>(test: T, mvt
 };
 
 export const selectAmountsTestVariant = (
-    test: ModifiedChoiceCardAmounts,
+    tests: AmountsTests,
+    countryCode: string,
     countryGroupId: CountryGroupId,
     mvtId: number,
 ): SelectedAmountsVariant | undefined => {
-    const regionTest = test[countryGroupId];
+    // Two-tier amounts - check for country test, then region test
+    let testArray = tests.filter(t => t.target === countryCode);
+    // If there is a country level test we need to discount it if its isLive boolean is false
```

Comment:
What do you think about doing this `isLive` check as part of the countryCode match above?

```
let testArray = tests.find(t => t.target === countryCode && t.isLive);
```

This feels more natural to me as it's doing the country matching logic in one place.

Reply:
I'll give it a go

Reply:
Done